.env.devnet

```diff
@@ -0,0 +1,105 @@
+# Deployer private key
+PRIVATE_KEY=0x
+
+# Network to deploy to (mainnet, testnet, or devnet)
+NETWORK="devnet"
+
+# Owner address for L1LiskToken contract
+L1_TOKEN_OWNER_ADDRESS=0x02cBF39018B40a48f0d95d3a0b2eeeF3D18e42Fe
+
+# Owner of L1VestingWallet contract
+L1_VESTING_WALLET_OWNER_ADDRESS=0x02cBF39018B40a48f0d95d3a0b2eeeF3D18e42Fe
+
+# Owner of L2Claim contract
+L2_CLAIM_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
+
+# Owner of L2Staking contract
+L2_STAKING_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
+
+# Owner of L2LockingPosition contract
+L2_LOCKING_POSITION_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
+
+# Owner of L2Governor contract
+L2_GOVERNOR_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
+
+# Owner of L2VotingPower contract
+L2_VOTING_POWER_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
+
+# Owner of L2Reward contract
+L2_REWARD_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
+
+# Owner of L2Airdrop contract
+L2_AIRDROP_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
+
+# Owner of L2VestingWallet contract
+L2_VESTING_WALLET_OWNER_ADDRESS=0x84277C9255b3B6904aCfe12C2c45c6a36E81B059
+
+# Airdrop wallet address where LSK tokens are transferred to after airdrop period ends
+L2_AIRDROP_WALLET_ADDRESS=0xaF3f4B5033Ef666b3F3EA91a0f360541cC21e5d7
+
+# Salt for deterministic smart contract address generation
+DETERMINISTIC_ADDRESS_SALT="lisk_l2_token_salt_test_network"
```

Comment:
I would not re-use the salt from testnet

Reply:
modified